script.pl

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/perl
+
+
+my %port = get_port();
+# my @names = ('Shuai Ni', 'Yifu Sun', 'Zhi Xu', 'Yang Tian', 'Wenji Liu', 'Shuai Hu', 'Fangshi Li', 'Feiyi Wang', 'Oulu Xu', 'Jing Li');
+my @names = ('Shuai Ni', 'Yifu Sun', 'Zhi Xu', 'Yang Tian');
+my $names_len = scalar @names;
+my $msgCount = 100;
+
+
+# Total message sent is $msgCount x Combination(n, 2)
+for (my $i = 0; $i < $names_len; $i += 1) {
```

Comment:
For performance test purpose I think we want to run all different receiver/sender combinations in parallel.
Bash has a single "parallel" command that can start the same program multiple times with different parameters, which we should be able to use.